### ISSUE TITLE
New at-file Path Code Did Not Open File In A Child Of @path

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#3270]
Opening an outline for the first time after the path-handling code changes, and external file was not found that does exist in the stated place.  The `@file` node is the child of an `@path` node:

```
-  External File Launcher
   - @path c:/temp/leo
       - @file launcher-test.jl       ----> "can not open C:/Users/tom/.leo/launcher-test.jl"
```

Notice that Leo made up the wrong path for this file.  It is in `c:/temp/leo`, **not** `C:/Users/tom/.leo/`.  This at-file node is in the workbook, which is in ~/.leo. It looks like the `@path` node is getting ignored.

[Later] May be the same bug as in https://github.com/leo-editor/leo-editor/issues/3272.

**Comment by edreamleo** [comment]
@tbpassin Thanks for this report!

`c.fullPath` calls `c.scanAtPathDirectives`.  The problem almost surely lies in the latter method.

**Comment by edreamleo** [comment]
Closed via PR #3275.
